### PR TITLE
394745 Helm update on frontends

### PR DESCRIPTION
**Pull request by rtasalem**
https://dev.azure.com/defragovuk/DEFRA-FFC/_workitems/edit/394745

### CODE REVIEW

File: appConfig/appConfig.snd3.yaml [hunk 1]
```diff
@@ -26,9 +26,9 @@
 - key: DEFRA_ID_POLICY
   value: "b2c_1a_cui_cpdev_signupsigninsfi"
 - key: DEFRA_ID_REDIRECT_URL
-  value: "https://ffc-sfd-landing-page.snd3.adp.defra.gov.uk/auth/sign-in-oidc"
+  value: "https://ffc-sfd.snd3.adp.defra.gov.uk/auth/sign-in-oidc"
```
Comment (rtasalem) [inline appConfig/appConfig.snd3.yaml:29]: Couldn't remember if removing `landing-page` from the URLs in the `appConfig` also applied to these two Defra ID URLs or if what only applicable to the `GATEWAY_HOST`? Let me know and I can undo these changes if I'm incorrect.

File: appConfig/appConfig.snd3.yaml [hunk 2]
```diff
@@ -26,9 +26,9 @@
...
 - key: DEFRA_ID_POST_LOGOUT_REDIRECT_URL
-  value: "https://ffc-sfd-landing-page.snd3.adp.defra.gov.uk/auth/sign-out-oidc"
+  value: "https://ffc-sfd.snd3.adp.defra.gov.uk/auth/sign-out-oidc"
```
Comment (rtasalem) [inline appConfig/appConfig.snd3.yaml:31]: Same as above comment.

File: appConfig/appConfig.snd3.yaml (hunk 1 above)
Comment (rtasalem) [reply appConfig/appConfig.snd3.yaml:29]: Ignore this. It does apply.

File: appConfig/appConfig.snd3.yaml (hunk 2 above)
Comment (rtasalem) [reply appConfig/appConfig.snd3.yaml:31]: Ignore this for the same reasons also.

**Comment by simonnineteen90** [comment]
Looks good but need to wait for update from DefraID and any ADP changes that are required. Also for the service to be stable again as currently not available.

**Comment by rtasalem** [comment]
> Looks good but need to wait for update from DefraID and any ADP changes that are required. Also for the service to be stable again as currently not available.

@simonnineteen90 Yep agreed, thanks for having a look over all the PRs.

**Comment by rtasalem** [comment]
Also noting here that the pod logs are deploying with no errors, but as the change in ingress is throwing an error on the deployed service itself will wait till Defra ID update the redirect URLs.

**Review by FayToward — APPROVED** [review]
LGTM

**Pull request merged**